### ISSUE TITLE
sequence tagging 任务预估时报  Segmentation fault

### ISSUE THREAD

**Issue by AoZhang** [PaddlePaddle/Paddle#1350]
具体日志如下：

```
I0216 19:12:13.527242 17281 GradientMachine.cpp:123] Loading parameters from output/model/pass-00009
[INFO 2017-02-16 19:12:13,539 dataprovider.py:236] num_sequences=90
I0216 19:12:15.992811 17281 Tester.cpp:257]  Pass=0 samples=90 AvgCost=0 Eval: error=346.554 chunk_f1=0.759546 true_chunks=500 result_chunks=469 correct_chunks=368
*** Aborted at 1487243535 (unix time) try "date -d @1487243535" if you are using GNU date ***
PC: @     0x7f206e5e3674 std::thread::join()
*** SIGSEGV (@0x0) received by PID 17281 (TID 0x7f206f28f780) from PID 0; stack trace: ***
    @     0x7f206ee69160 (unknown)
    @     0x7f206e5e3674 std::thread::join()
    @           0x6d6e6c paddle::DoubleBuffer::~DoubleBuffer()
    @           0x6d70a1 paddle::DoubleBuffer::~DoubleBuffer()
    @           0x6c5051 paddle::PyDataProvider2::~PyDataProvider2()
    @           0x59dfb9 std::_Sp_counted_base<>::_M_release()
    @           0x59e8bf paddle::Trainer::~Trainer()
    @           0x591f6d main
    @     0x7f206dc8ebd5 __libc_start_main
    @           0x59dbf5 (unknown)
/***/paddle/output/bin/paddle_local: line 109: 17281 Segmentation fault      ${DEBUGGER} $MYDIR/../opt/paddle/bin/paddle_trainer ${@:2}
```

predict的网络配置跟sequence_tagging demo里的
[rnn_crf.py](https://github.com/PaddlePaddle/Paddle/blob/develop/demo/sequence_tagging/rnn_crf.py
)完全一致（部分参数有修改），只是最后的输出改成了
```
outputs(crf_decoding)
```

**Comment by AoZhang** [comment]
补充一些信息：

预估的启动命令如下，
```
cfg=rnn_crf_test.py
model="output/model/pass-00009"
paddle train \
    --config=$cfg \
    --job=test \
    --init_model_path=$model \
    --config_args=is_predict=1 \
    --predict_output_dir=output.predict
```
任务段错误退出后， output.predict 目录里已经有输出结果了。

**Comment by qingqing01** [comment]
@AoZhang 你的测试结果没有问题，这个是析构的错误，随后我们fix下，不影响测试结果吧~

**Comment by AoZhang** [comment]
看crf_decoding_layer的说明：

> A layer for calculating the decoding sequence of sequential conditional random field model. The decoding sequence is stored in output.ids. If a second input is provided, it is treated as the ground-truth label, and this layer will also calculate error. output.value[i] is 1 for incorrect decoding or 0 for correct decoding.

有个output.ids和output.value，请问这两组值分别如何输出？

我上面测试的输出**只有一列数据0/1**，并没有具体的预估label（也就是那个output.ids）。如果我想得到具体的label，请问需要如何配置？
